helm_deploy/hmpps-interventions-service/templates/_envs.tpl

```diff
@@ -16,4 +16,28 @@ env:
   - name: HMPPSAUTH_BASEURL
     value: "{{ .Values.env.HMPPSAUTH_BASEURL }}"
 
+  - name: POSTGRES_URI
+    valueFrom:
+      secretKeyRef:
+        name: postgres
+        key: rds_instance_endpoint
```

Comment:
Could `POSTGRES_URI` point to the [`url`](https://github.com/ministryofjustice/cloud-platform-environments/blob/1d3a8eed05977fd71bcd09272b7eb507c6cf5817/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/rds.tf#L33
) variable? That way we could save the db/username/password variables.

Reply:
Ah okay, I can see it's used in constructing a URL
```
 21     driver-class-name: org.postgresql.Driver
 22     url: jdbc:postgresql://${postgres.uri}/${postgres.db}
 23     username: ${postgres.username}
 24     password: ${postgres.password}
```

Reply:
`postgres.uri` has connotations of [`URI = scheme:[//authority]path[?query][#fragment]`](https://tools.ietf.org/html/rfc3986#section-1.1.1), which confused me